src/Components/Scheduling/renderHelper.tsx

```diff
@@ -66,6 +66,8 @@ const placeBlocks = <DataCourseBlock extends CourseBlock>(data: DataCourseBlock[
   const startsDuring = (block: DataCourseBlock) => block.start_time > region_start && block.start_time < region_end;
   const endsDuring = (block: DataCourseBlock) => block.end_time < region_end && block.end_time > region_start;
   const isDuring = (block: DataCourseBlock) => block.start_time < region_start && block.end_time > region_end;
+
+  // const collides = (block: DataCourseBlock) => !(block.end_time < region_start || block.start_time > region_end)
```

Comment:
Would remove this if there's no purpose for it anymore; if there still is a purpose then all good.

Reply:
It's Dante's, it has potential for optimization later so I think I'm going to leave it in just in case